v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -232,6 +232,32 @@ Please double check your arguments.`,
 
 Please double check your arguments.`,
     },
+    UNRECOGNIZED_NAMED_PARAM: {
+      number: 304,
+      messageTemplate:
+        "Invalid parameter {parameter}. It is neither a valid global parameter nor associated with any task. Did you forget to add the task first, or did you misspell it?",
```

Comment:
This error is slightly confusing. It's not associated with the current task, not with any task.

Reply:
Related to this comment: https://github.com/NomicFoundation/hardhat/pull/5282#discussion_r1622408255